<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/myicon/image/background.gif</normaloff>:/myicon/image/background.gif</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QLineEdit" name="lineEditIP">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>9</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Server_IP：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend_shaft_01_Left">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/myicon/actions/gtk-go-back-ltr.png</normaloff>:/myicon/actions/gtk-go-back-ltr.png</iconset>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditRead">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>35</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnect">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditWrite">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend_shaft_04_Left">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/myicon/actions/gtk-go-back-ltr.png</normaloff>:/myicon/actions/gtk-go-back-ltr.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend_shaft_02_Left">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/myicon/actions/gtk-go-back-ltr.png</normaloff>:/myicon/actions/gtk-go-back-ltr.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend_shaft_05_Left">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/myicon/actions/gtk-go-back-ltr.png</normaloff>:/myicon/actions/gtk-go-back-ltr.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend_shaft_03_Left">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/myicon/actions/gtk-go-back-ltr.png</normaloff>:/myicon/actions/gtk-go-back-ltr.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend_shaft_06_Left">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/myicon/actions/gtk-go-back-ltr.png</normaloff>:/myicon/actions/gtk-go-back-ltr.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>shaft_1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>shaft_2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>shaft_3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>shaft_4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend_shaft_01_Right">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/myicon/actions/gtk-go-back-rtl.png</normaloff>:/myicon/actions/gtk-go-back-rtl.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend_shaft_02_Right">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/myicon/actions/gtk-go-back-rtl.png</normaloff>:/myicon/actions/gtk-go-back-rtl.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend_shaft_03_Right">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/myicon/actions/gtk-go-back-rtl.png</normaloff>:/myicon/actions/gtk-go-back-rtl.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend_shaft_04_Right">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/myicon/actions/gtk-go-back-rtl.png</normaloff>:/myicon/actions/gtk-go-back-rtl.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend_shaft_05_Right">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/myicon/actions/gtk-go-back-rtl.png</normaloff>:/myicon/actions/gtk-go-back-rtl.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend_shaft_06_Right">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/myicon/actions/gtk-go-back-rtl.png</normaloff>:/myicon/actions/gtk-go-back-rtl.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReset">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Speed">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCrawl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Crawl</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlace">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Place</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>shaft_5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>shaft_6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
